std/assembly/string.ts

```diff
@@ -380,6 +386,31 @@ export class String {
     );
     return out;
   }
+
+  repeat(count: i32 = 0): String {
+    assert(this !== null);
+    assert(count >= 0);
```

Comment:
I believe JS does `throw new RangeError("Invalid count value")` there, hmm. Still unsure what's best as this will eventually result in quite a few static strings in modules where methods that do so are used. For now I'd tend to stay consistent with JS here, as other functions do it as well.